src/main/java/org/tomitribe/auth/signatures/Signatures.java

```diff
@@ -33,7 +58,31 @@ public static String createSigningString(final List<String> required, String met
             if ("(request-target)".equals(key)) {
                 method = lowercase(method);
                 list.add(Join.join(" ", "(request-target):", method, uri));
-
+            } else if ("(created)".equals(key)) {
+                // The "created" parameter contains the signature's Creation Time.
+                // This parameter is useful when signers are not capable of controlling
+                // the "Date" HTTP Header such as when operating in certain web
+                // browser environments.
+                // Its canonicalized value is an Integer String containing the
+                // signature's Creation Time expressed as the number of seconds since
+                // the Epoch
+                if (signatureCreationTime == null) {
+                    throw new InvalidCreatedFieldException("(created) field requested but signature creation time is not set");
+                }
+                list.add(key + ": " + Long.toString(signatureCreationTime / 1000L));
```

Comment:
For this kind of thing, TimeUnit allows to quickly switch from one unit to the other.
A detail only

Reply:
done